app/celer-g4/RootIO.cc

```diff
@@ -62,76 +62,76 @@ HitRootIO::HitRootIO()
 
         file_.reset(TFile::Open(file_name_.c_str(), "recreate"));
         CELER_VALIDATE(file_->IsOpen(), << "failed to open " << file_name_);
-        tree_.reset(new TTree(
-            "Events", "Hit collections", this->SplitLevel(), file_.get()));
+        tree_.reset(new TTree(this->TreeName(),
+                              this->TreeName(),
```

Comment:
This should be a tile - of course same as the name, but why is the point dropping/degrading the explicit title here?

Reply:
Just because we have to set up the same tree name in 2 different functions (`RootIO` and `Merge`). So I unified the name in a single place. Easier to maintain if we want to update it as well.